app/models/node.rb

```diff
@@ -1,14 +1,12 @@
 class UniqueUrlValidator < ActiveModel::Validator
   def validate(record)
-    if record.title == '' || record.title.nil?
-      # record.errors[:base] << "You must provide a title."
+    if record.title.blank?
+      record.errors[:base] << "You must provide a title."
       # otherwise the below title uniqueness check fails, as title presence validation doesn't run until after
```

Comment:
Oddly, tests are passing whether or not the title is checked for blankness!

Reply:
Is this really needed? Isn't it covered by:
`validates title, presence: true, ....` (on line 97)